client/landing/gutenboarding/domain-suggestions/resolvers.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+import { apiFetch } from '@wordpress/data-controls';
```

Comment:
There's a bit of a nuance on this one, since this isn't actually `apiFetch from '@wordpress/api-fetch'`, but, well, `from '@wordpress/data-controls'`. The latter adds some (required) indirection for [`controls`](https://developer.wordpress.org/block-editor/packages/packages-data/#controls) -- basically listeners that perform certain (async) side effects in response to a certain action being fired.

Most of `{ apiFetch } from '@wordpress/data-controls'` is fairly simple boilerplate, but it's tightly coupled to `@wordpress/api-fetch` -- if we wanted to use regular `fetch`, we'd have to write our own `controls`. Not a big tradeoff, just pointing it out; and I'm not 100% sure it's worth the effort, if `apiFetch` also works (and is a bit more WordPress-y).